### PR TITLE
Added mcmillan_airfield model and example world

### PR DESCRIPTION
**Pull request by ahcorde**
Added mcmillan_airfield 
 - model
 - example world

mcmillan

co-author: @tfoote

### CODE REVIEW

**Comment by tfoote** [comment]
Please test this out as is for feedback.  Before it's ready to merge we're going to move the models to be hosted at gazebosim.org/models so that the model data isn't stored here in this repository. But before we finalize the model upload there we'd like to get any feedback from reviews here.

**Comment by ahcorde** [comment]
```
    os.environ["PX4_HOME_LAT"] = "35.7222542";
    os.environ["PX4_HOME_LON"] = "-120.7665102";
    os.environ["PX4_HOME_ALT"] = "280";
```

File: worlds/mcmillan_airfield.world [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        </clouds>
+      </sky>
+    </scene>
+    <!-- A global light source -->
+    <include>
+      <uri>model://sun</uri>
+    </include>
+    <include>
+      <uri>model://asphalt_plane</uri>
+    </include>
```
Comment (Jaeyoung-Lim) [inline worlds/mcmillan_airfield.world]: Could you maybe remove the asphalt plane if it isn't necessary?

**Review by Jaeyoung-Lim — CHANGES_REQUESTED** [review]

File: worlds/mcmillan_airfield.world (hunk 1 above)
Comment (ahcorde) [reply worlds/mcmillan_airfield.world]: :+1:

**Review by TSC21 — APPROVED** [review]

**Comment by Jaeyoung-Lim** [comment]
Screenshot from 2020-03-26 12-22-20(1)

I tested the world but the terrain seems to be above the origin. @ahcorde was this also the case for you?

**Comment by Jaeyoung-Lim** [comment]
This needs to be fixed:

**Comment by tfoote** [comment]
@Jaeyoung-Lim That's expected. The model is at our best approximation of actual altitude.

**Comment by tfoote** [comment]
Maybe there's a disconnect between when things were translated to the xml declaration from the PX4_HOME_ALT setting we were testing with previously. Should this read 0?

 https://github.com/PX4/sitl_gazebo/pull/406/files#diff-5c17f02d16a0b9461d241f204548a4dfR22

**Comment by TSC21** [comment]
> Maybe there's a disconnect between when things were translated to the xml declaration from the PX4_HOME_ALT setting we were testing with previously. Should this read 0?
> 
> https://github.com/PX4/sitl_gazebo/pull/406/files#diff-5c17f02d16a0b9461d241f204548a4dfR22

@ahcorde

**Pull request merged**